functions/mqhandlers.go

```diff
@@ -585,6 +618,7 @@ func mqFallback(ctx context.Context, wg *sync.WaitGroup) {
 			}
 			// Call netclient http config pull
 			slog.Info("### mqfallback routine execute")
+			auth.CleanJwtToken()
```

Comment:
why clear jwt token?

Reply:
clear jwt token to force call authenticate.
In authenticate in server side, if the client id is not in emqx, it will create a new client id. 

There was an issue found during the test. After the emqx restart, sometimes(not always, not for every client), some of the client ids were missed in emqx.  In the situation, trying to connect to emqx returned "not authed" error. 
The fix is to add the client id back if it's not there.